tests/test_fairstep.py

```diff
@@ -273,6 +274,11 @@ def test_step(a:float, b:float) -> float:
     else:
         raise
 
+    assert test_step._fairstep.language is not None
+    assert isinstance(test_step._fairstep.language, LinguisticSystem)
+    assert 'python' in test_step._fairstep.language.label
```

Comment:
Maybe we should also check that some of the triples are actually in the RDF?

Reply:
Agreed. I've added a few checks for triples that should be in the `_rdf`